src/resultset/formatters/resultset_replyverbose.c

```diff
@@ -153,6 +159,12 @@ static void _ResultSet_VerboseReplyWithArray(RedisModuleCtx *ctx, SIValue array)
 	rm_free(str);
 }
 
+static void _ResultSet_VerboseReplyWithPath(RedisModuleCtx *ctx, SIValue path) {
+	SIValue path_array = SIPath_ToList(path);
+	_ResultSet_VerboseReplyWithArray(ctx, path_array);
```

Comment:
This is just a personal comment, as I think this has already gone through design, but our current returns look like:
```
GRAPH.QUERY social "MATCH p = (a)-[e]->(b) RETURN p LIMIT 2"
1) 1) "p"
2) 1) 1) "[(2), [18], (1)]"
   2) 1) "[(2), [16], (5)]"
3) 1) "Query internal execution time: 1.053858 milliseconds"
```
I feel like toString calls would be more informative, if that is possible! We can revisit this later.